sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ClientSideRequestStatistics.java

```diff
@@ -275,8 +264,8 @@ public void serialize(ClientSideRequestStatistics statistics, JsonGenerator gene
             generator.writeStartObject();
             long requestLatency = statistics.getRequestLatency().toMillis();;
             generator.writeNumberField("requestLatency", requestLatency);
-            generator.writeStringField("requestStartTime", formatDateTime(statistics.requestStartTime));
-            generator.writeStringField("requestEndTime", formatDateTime(statistics.requestEndTime));
+            generator.writeStringField("requestStartTimeUTC", ZonedDateTimeSerializer.formatDateTime(statistics.requestStartTimeUTC));
```

Comment:
I noticed in the resultant diagnostics string some fields are utc and some have timezone. I wonder why you have the mix of the two?

Reply:
trying to move every time to utc to avoid confusion during debugging